thundra/listeners/composite_span_filter.py

```diff
@@ -0,0 +1,26 @@
+from thundra.listeners.thundra_span_filterer import SpanFilter
+
+
+class CompositeSpanFilter(SpanFilter):
+    
+    def __init__(self, composite=True, is_all=False, filters=[]):
+        self.composite = composite
+        self.all = is_all
+        self.filters = filters
+
+    def accept(self, span):
+        if (not self.filters) or len(self.filters) == 0:
+            return True
+
+        result = self.all
```

Comment:
when there is no filter (filters are empty), result is different for `is_all` is `true` and `false` values